package.json

```diff
@@ -1,40 +1,59 @@
 {
-  "name": "is-valid-itin",
-  "version": "0.0.2",
-  "description": "Validate Individual Taxpayer Identification Number",
-  "main": "dist/index.js",
-  "homepage": "https://github.com/seegno/is-valid-itin",
-  "bugs": "https://github.com/seegno/is-valid-itin/issues",
-  "repository": "seegno/is-valid-itin",
-  "author": "Seegno",
-  "license": "MIT",
+  "name": "itin-validator",
+  "version": "1.0.0",
+  "description": "Individual Taxpayer Identification Number validator and masker",
   "keywords": [
     "IRS",
-    "TIN",
-    "ITIN"
+    "ITIN",
+    "US",
+    "USA",
+    "identification",
+    "taxpayer",
+    "validator"
   ],
-  "options": {
-    "mocha": "--compilers js:babel-register --recursive --require should"
+  "homepage": "https://github.com/seegno/itin-validator",
+  "bugs": "https://github.com/seegno/itin-validator/issues",
+  "license": "MIT",
+  "author": "Seegno",
+  "main": "dist/index.js",
```

Comment:
src/dist even if we just transpire the src for now? Allows including test in the future.